### ISSUE TITLE
custom.css?

### ISSUE THREAD

**Issue by kakila** [petrnohejl/minimal-xy#6]
The example configuration points to custom.css which is not part of the theme... do we need to create that file?

**Comment by sahil-sagwekar2652** [comment]
Yes, I would think so. It's for the user to add customization on top of the theme.
